pkg/run/run.go

```diff
@@ -122,12 +134,12 @@ func (r *Runtime) Run() error {
 			return r.Stop(nil)
 
 		case <-timeoutTimer.C:
-			glog.Warningf("Timeout %s reached, stopping ...", r.runTimeout.String())
-			return r.Stop(fmt.Errorf("timeout reached during run phase: %s", r.runTimeout.String()))
+			glog.Warningf("Timeout %s reached, stopping ...", r.conf.RunTimeout.String())
```

Comment:
You don't need to call `String()` when using the format specifier `%s`.

Reply:
Yes but it's painful to change it everywhere. I like to keep it consistent.
Should I change the formatter by `%v` ?

Reply:
I've never really been able to find a clear convention for when to use `%s` instead of `%v`.

I'll leave it up to you, but I don't think `String()` should ever be called directly if you can help it.